gnucash/gnome-utils/window-main-summarybar.c

```diff
@@ -412,6 +408,10 @@ gnc_main_window_summary_refresh (GNCMainSummary * summary)
                              currency_accum->profits,
                              gnc_commodity_print_info(currency_accum->currency, TRUE));
 
+            mnemonic = gnc_commodity_get_nice_symbol (currency_accum->currency);
+            if (mnemonic == NULL)
+                mnemonic = "";
+
```

Comment:
I have an even better idea: `mnemonic` is used only once, as is `get_total_mode_label`, so move the `gnc_commodity_get_nice_symbol` call into `get_total_mode_label` and remove its `mnemonic` parameter.

Reply:
Merged in. There's still the risk of another call to `gnc_commodity_get_nice_symbol` somewhere else and the old static char* is still kept longer than allowed.

Reply:
There's no risk in this case because it now retrieves the symbol and immediately passes it to the widget where it's copied. No doubt there are other places where that's not the case.